### ISSUE TITLE
Chairs and stools say "drag your sprite to sit in it" which doesn't work

### ISSUE THREAD

**Issue by cacogen** [tgstation/tgstation#49778]
Issue reported from  Round ID: 131669 (/tg/Station Manuel [ENGLISH] [US-WEST] [ROLEPLAY] [99% LAG FREE])

Reporting client version: 513.1508

I think this means buckling, but buckling only works when you're on the same turf as the chair and stools can't be buckled to at all.

**Comment by Timberpoes** [comment]
While a very minor bug, this was still a bug. I have fixed this in #49827 for you.

You were correct to assume that sitting meant buckling. Stools very specifically have a piece of code that means they can't be buckled to by design. I've added some extra logic to handle this edge case and any pseudo-chairs that can't be buckled to will not output the sprite dragging text when examined.

Because you specifically mentioned buckling only works when on the same turf as the chair - As if to say that you expected to be able to move into the chair from the tile next to it, I also added a bit of extra detail in the examine text explaining how the player must be stood on the same tile as the chair in order to drag their sprite and sit. I hope this addresses both parts of your issue.
